src/components/ServiceDetails/AboutService/ServiceDemo/ThirdPartyAIService.js

```diff
@@ -137,6 +143,7 @@ const mapStateToProps = state => ({
   grpcResponse: state.serviceReducer.serviceMethodExecution.response,
   isComplete: state.serviceReducer.serviceMethodExecution.isComplete,
   username: state.userReducer.username,
+  serviceMetadata: state.serviceReducer.serviceMetadata,
```

Comment:
You might want to have `serviceMetadata` associated to a particular `service` and fetch the specific service metadata.

Reply:
[waiting for service metadata in the parent component](https://github.com/singnet/refactored-dapp/pull/127/commits/f3e923a9876750b8cd217e1d67bd792657e952bb)

Reply:
isolated serviceDetails as a separateReducer.
[commit](https://github.com/singnet/refactored-dapp/pull/127/commits/2ea065e373c98c7c1ded24daf3c31d0047ebae99)
[commit](https://github.com/singnet/refactored-dapp/pull/127/commits/92d9a116a6f20cf0fe373c9e74c9b6d24280eda4)

> Did not move serviceMetthodExecution because we would get rid of it once sdk is integrated to all the AIs